Update mendix-feedback.md

mel-chiu opened the pull request:
A new simplified feedback widget document

mel-chiu committed:
Update mendix-feedback.md

A new simplified feedback widget document

mel-chiu committed:
Update mendix-feedback.md

mel-chiu committed:
Update 1.3 Limitations 

Added limitation for the Native Feedback widget.

mel-chiu committed:
Updated 1.3 Limitations links

Luyao-Zhang-1 committed:
Finish the inital review

mel-chiu commented on content/en/docs/appstore/widgets/mendix-feedback.md:
```diff
@@ -11,119 +11,116 @@ tags: ["marketplace", "marketplace component", "widget", "feedback", "platform s
+  
+    *  **App ID** – the unique identifier of your app, which you can find in your app’s [General Settings](/developerportal/collaborate/general-settings/) in the Developer Portal
+    
+        {{% alert color="info" %}}The original value of **App ID** is 1, but this value should automatically change to your correct app ID. if it does not change automatically, see [Updating App ID](update-app-id) in the Troubleshooting section.{{% /alert %}}
```
Do we need a # in front of update-app-id? -> `#update-app-id`

mel-chiu commented on content/en/docs/appstore/widgets/mendix-feedback.md:
```diff
@@ -11,119 +11,116 @@ tags: ["marketplace", "marketplace component", "widget", "feedback", "platform s
+    
+        {{% alert color="info" %}}The original value of **App ID** is 1, but this value should automatically change to your correct app ID. if it does not change automatically, see [Updating App ID](update-app-id) in the Troubleshooting section.{{% /alert %}}
+        
+        **Allow screenshots**
```
Maybe we can explain the function of `allow screenshots` here? 
For example:
**Allow screenshots** - Control whether the app user could take a screenshot

Luyao-Zhang-1 committed:
Add # to achor

Luyao-Zhang-1 committed:
Add the description of "Allow screenshots"

Luyao-Zhang-1 committed:
Move images to correct location

Luyao-Zhang-1 commented:
@mel-chiu I will be on vacation from next Monday till May 26. If you need to update or release this doc before I am back, please contact my teammate @ConnorLand . Thanks!

mel-chiu committed:
Improved troubleshooting section

-Added 2 extra troubleshooting points.
-Fixed typos
-Added some detailed explanations.

ConnorLand committed:
Proofread and format

Made some formatting changes to bring the lists more in line with conventions.
